AFQ/api/group.py

```diff
@@ -916,6 +916,89 @@ def assemble_AFQ_browser(self, output_path=None, metadata=None,
             sublink=page_subtitle_link)
 
 
+class ParallelGroupAFQ(GroupAFQ):
+    def __init__(self, *args, **kwargs):
+        orig = GroupAFQ(*args, **kwargs)
+
+        if "submitter_params" not in orig.parallel_params:
+            orig.parallel_params["submitter_params"] = {"plugin": "cf"}
+
+        if "cache_dir" not in orig.parallel_params:
+            orig.parallel_params["cache_dir"] = None
+
+        self.parallel_params = orig.parallel_params
+        self.pAFQ_kwargs = [pAFQ.kwargs for pAFQ in orig.pAFQ_list]
+
+        # Rename kwargs and clear "bids_info" and "base_fname"
+        # ParticipantAFQ takes in these parameters under one name but stores
+        # and uses them under another
+        for ii in range(len(self.pAFQ_kwargs)):
+            self.pAFQ_kwargs[ii]["dwi_data_file"] = \
+                self.pAFQ_kwargs[ii]["dwi_path"]
+
+            self.pAFQ_kwargs[ii]["bval_file"] = \
+                self.pAFQ_kwargs[ii]["bval"]
+
+            self.pAFQ_kwargs[ii]["bvec_file"] = \
+                self.pAFQ_kwargs[ii]["bvec"]
+
+            self.pAFQ_kwargs[ii]["output_dir"] = \
+                self.pAFQ_kwargs[ii]["results_dir"]
+
+            del self.pAFQ_kwargs[ii]["dwi_path"]
+            del self.pAFQ_kwargs[ii]["bval"]
+            del self.pAFQ_kwargs[ii]["bvec"]
+            del self.pAFQ_kwargs[ii]["results_dir"]
+            del self.pAFQ_kwargs[ii]["bids_info"]
+            del self.pAFQ_kwargs[ii]["base_fname"]
+
+    def export_all(self, viz=True, afqbrowser=True, xforms=True, indiv=True):
+        """ Exports all the possible outputs
+
+        Parameters
+        ----------
+        viz : bool
+            Whether to output visualizations. This includes tract profile
+            plots, a figure containing all bundles, and, if using the AFQ
+            segmentation algorithm, individual bundle figures.
+            Default: True
+        afqbrowser : bool
+            Whether to output an AFQ-Browser from this AFQ instance.
+            Default: True
+        xforms : bool
+            Whether to output the reg_template image in subject space and,
+            depending on if it is possible based on the mapping used, to
+            output the b0 in template space.
+            Default: True
+        indiv : bool
+            Whether to output individual bundles in their own files, in
+            addition to the one file containing all bundles. If using
+            the AFQ segmentation algorithm, individual ROIs are also
+            output.
+            Default: True
+        """
+        import pydra
```

Comment:
I'd suggest adding: 

```
from dipy.utils.optpkg import optional_package

pydra, has_pydra, _ = optional_package('pydra')
``` 

At the top of the file for the pydra import, instead of importing within the function.